test/test_matmul_sass.cpp

```diff
@@ -206,121 +209,251 @@ TEST_F(MatmulSASSTest, AmpereModifiers_CUDA) {
-                  const std::vector<std::string> expect;
-                  TORCH_CHECK(
-                      i.modifiers() == expect,
-                      "Modifiers for LDGDEPBAR has changed. "
-                      "Please manually check if the new modifiers makes sense and update this test. "
-                      "Expect: ",
-                      expect,
-                      " Get: ",
-                      i.modifiers());
-                  found_LDGDEPBAR = true;
-                } else if (i.opCode() == "LDSM") {
-                  const std::vector<std::string> expect1 = {"16", "M88", "2"};
-                  const std::vector<std::string> expect2 = {"16", "M88", "4"};
-                  const std::vector<std::string> expect3 = {"16", "MT88", "2"};
-                  const std::vector<std::string> expect4 = {"16", "MT88", "4"};
-                  TORCH_CHECK(
-                      i.modifiers() == expect1 || i.modifiers() == expect2 ||
-                          i.modifiers() == expect3 || i.modifiers() == expect4,
-                      "Modifiers for LDGDEPBAR has changed. "
-                      "Please manually check if the new modifiers makes sense and update this test.");
-                  found_LDSM = true;
-                } else if (i.opCode() == "HMMA") {
-                  const std::vector<std::string> expect = {"16816", "F32"};
-                  TORCH_CHECK(
-                      i.modifiers() == expect,
-                      "Modifiers for HMMA has changed. "
-                      "Please manually check if the new modifiers makes sense and update this test. "
-                      "Expect: ",
-                      expect,
-                      " Get: ",
-                      i.modifiers());
-                  found_HMMA = true;
-                } else if (i.opCode() == "BAR") {
-                  const std::vector<std::string> expect = {
-                      "SYNC", "DEFER_BLOCKING"};
-                  TORCH_CHECK(
-                      i.modifiers() == expect,
-                      "Modifiers for BAR has changed. "
-                      "Please manually check if the new modifiers makes sense and update this test. "
-                      "Expect: ",
-                      expect,
-                      " Get: ",
-                      i.modifiers());
-                  BAR_COUNT++;
-                } else if (i.opCode() == "DEPBAR") {
-                  const std::vector<std::string> expect = {"LE"};
-                  TORCH_CHECK(
-                      i.modifiers() == expect,
-                      "Modifiers for DEPBAR has changed. "
-                      "Please manually check if the new modifiers makes sense and update this test. "
-                      "Expect: ",
-                      expect,
-                      " Get: ",
-                      i.modifiers());
-                  found_DEPBAR = true;
-                }
+  bool found_LDGSTS = false;
```

Comment:
Reversed the change to `AmpereModifiers_CUDA` and seperate the use_smem_epilogue test to a new test.